azure-pipelines.lgtm.yml

```diff
@@ -26,12 +26,15 @@ variables:
   value: true
 - name: buildConfiguration
   value: Release
+- name: msbuild
+  value: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\msbuild.exe'
```

Comment:
I saw this warning:

windows-latest pipelines will use windows-2022 soon. For more details, see actions/virtual-environments#4856

If windows-latest pipelines switch to windows-2022, is this path still exists?

Reply:
Aha, the naming with -2022 is ambiguous. Msbuild tool is actually not affected as VS2022 won't be installed on windows-2022. 

However, the post(https://github.com/actions/virtual-environments/issues/4856) mentioned that CodeQL wouldn't be supported on windows-2022. So we have to explicitly choose windows-2019.
![image](https://user-images.githubusercontent.com/59190910/150930338-05910c3f-9ca1-438c-94d8-58123ef02497.png)